data/templates/template.typst

```diff
@@ -35,55 +35,50 @@
     paper: paper,
     margin: margin,
     numbering: "1",
-  )
+    columns: cols,
+    )
+
```

Comment:
why the blank line?  we don't have one elsewhere, e.g. btw 33 and 34

Reply:
it was somewhat of an aid for me to debug the code; didn't seem to do any harm, but now I see the underlying rationales.

Reply:
it was there, in the previous version, between the page settings and the titlepage code.

Reply:
actually, it was there in the first place, it was just somewhat hidden from view in the change file...